qfast/instantiation/native/qs.py

```diff
@@ -25,7 +24,7 @@ def get_maximum_size ( self ):
 
         # Larger unitaries can be decomposed with this tool,
         # however, solution quality is best at 3 qubits.
-        return 3
+        return 4
```

Comment:
idk if this is a good idea, but I'm pretty confident LEAP can handle any 4 qubit unitary (it might just be a bit slow in the worst of cases)

Reply:
I have tested LEAP and it works well with 4, however, this number is best at 3 when integrating into QFAST's hierarchy.